docs/mobile-apps/features/video-recording.md

```diff
@@ -0,0 +1,42 @@
+---
+id: video-recording
+title: Video Recording
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Sauce Labs captures a recording of the device screen during your tests. For most test frameworks, this video becomes available upon test completion or as a live stream while the test is in progress.
+
+<p><small><span className="sauceGreen">Real Devices Documentation</span></small></p>
+
+For testing on real devices, video recording is done in one of two ways:
+- On-device recording
+  - Provides higher video quality and framerate, but requires an agent app running on the device
+- External screenshot-based recording
+  - Lower quality and framerate, but requires less set-up on the device
+The method of recording is decided automatically at the start of each test, based on device support (with a preference towards on-device recording).
+
+Once your test job concludes, the recorded video is encoded, downloaded from the device and uploaded to our long-term storage. Be aware that videos are automatically deleted after 30 days, or upon explicit deletion of the associated job.
+
+A test job will not be marked as finished until the video has been successfully uploaded to long-term storage. Consequently, very long-running test sessions will experience a more extended post-processing phase, thus increasing the overall duration of the job. A long session also makes it more likely that the video will either be missing or corrupted. We therefore recommend to limit the session duration, if the video is important to you.
+
+## Live Stream
+
+<p><small><span className="sauceGreen">Real Devices Documentation</span></small></p>
+
+While your test job is active, you can access a live video stream of the device if you have the job's device_session_id. This live stream is accessible via WebSocket at the following URL:
+
+`wss://api.<sauce_region>.saucelabs.com/v1/rdc/socket/alternativeIo/<device_session_id>`
+
+Access to this stream requires Basic Auth.
+
+Be aware that this is a raw data stream and requires parsing to be viewable. The video stream adheres to the [OpenSTF](https://github.com/openstf/minicap?tab=readme-ov-file#usage) standard. Each message that you receive on this web socket represents a screenshot taken on the device.
```

Comment:
I'm not sure it's any beneficial to mention OpenSTF unless it's for search engine optimisation. Also OpenSTF it is not a standard. I believe we might want to specify that it's an mjpeg stream instead.

Reply:
updated, removed the openstf description, since we are using a modified version of it. 

Especially the way that we are handling device input is custom and is not represented in openSTF.

We will have to specify this in more detail once we are opening up our APIs.